src/commands/read.ts

```diff
@@ -16,6 +16,13 @@ export const command = "read [statement]";
 export const desc = "Run a read-only query against a remote table";
 export const aliases = ["r", "query", "q"];
 
+function transformTableData(obj: any) {
```

Comment:
Rather than transforming the response, I think you can just use "all" versus "raw" for the SDK calls and you'll just get back the right format. Having said that, with "raw", I think it doesn't include the columns...

Reply:
You are correct; `raw` only includes rows with no column names, and `all` returns each row as an object, which is why it needs to be transformed. There is not a way to get this format:
```
{ 
columns: [...],
rows: [...],
}
```

Reply:
I'm good removing this transformation; I think it would be better to hew closer to the SDK than what we have in the docs currently.

Reply:
So just use "raw" and don't return the columns? Seems ok. Also solve the above question about returning the full response or not.